crates/swc_css_lints/src/pattern.rs

```diff
@@ -0,0 +1,41 @@
+use regex::Regex;
+
+#[derive(Debug)]
+pub(crate) enum NamePattern {
+    Str(String),
+    Regex(Regex),
```

Comment:
Can you use [CachedRegex](https://docs.rs/swc_cached/latest/swc_cached/regex/struct.CachedRegex.html) instead?

Reply:
I made `CachedRegex` because the configuration is typically shared among files in a project.